back_end/RayTracer/Models/Cameras/Perspective.cs

```diff
@@ -75,24 +80,28 @@ public override void Render(Scene scene)
             Ray ray = new Ray();
             ray.origin = position;
 
```

Comment:
Change the names of variables, and the order of the loops, to be traversing the image by using the height in the outer loop, and the width in the inner one. The height represents the row, while the width represents the column.